.dockerignore

```diff
@@ -1,4 +1,3 @@
-**/*
 !Makefile
 !*.c
```

Comment:
Ideally, the fix would be to change these to `!**/*.c` and `!**/*.h` rather than removing the ignore-all (`**/*`). Right now, this patch is effectively deleting the `.dockerignore` file entirely.

However, the project is so small, so not ignoring anything doesn't have much performance impact.